spec/requests/article_spec.rb

```diff
@@ -0,0 +1,196 @@
+require 'rails_helper'
+
+RSpec.describe "Articles", type: :request do
+  let(:initial_attributes) do
+    { title: 'test', body: 'test body'}
+  end
+
+  let(:valid_attributes) do
+    { title: 'new test', body: 'new body' }
+  end
+
+  let(:nil_title_attributes) do
+    { title: nil, body: 'new body' }
+  end
+
+  let(:nil_body_attributes) do
+    { title: 'new title', body: nil }
+  end
```

Comment:
Do we even need this? The controller doesn't have the responsibility of figuring out what things are going to make the `Article.save` fail

Reply:
Removed